### PR TITLE
Expandable box

### PR DESCRIPTION
**Pull request by o-t-w**

### CODE REVIEW

**Comment by alexkilgour** [comment]
Does this have an actual use case for one of our projects?
Also, you don't have to have the `scss` and `view` folders if this is a js component

**Comment by o-t-w** [comment]
Its used on Nature. I just wanted a js thing to use in the Vertical Slice.

File: packages/global-expandable-box/js/index.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+	} else {
+		open(box, closetext);
+		box.isOpen = true;
+	}
+}
+
+function init(boxes, height = '95px', closetext = 'Show less', opentext = 'Show more') {
+	boxes.forEach(box => {
+		let buttonEtc = '<div><div></div><div><button type="button" aria-expanded="false"><span>Show more</span></button></div></div>';
+		box.insertAdjacentHTML('besforeEnd', buttonEtc);
```
Comment (joeyciechanowicz) [inline packages/global-expandable-box/js/index.js]: `'besforeEnd' => 'beforeend'`

**Comment by alexkilgour** [comment]
We should have full details of how to use this in the `README.md`

**Comment by alexkilgour** [comment]
Your build is failing. You can use `npm run validate` to check issues locally

File: packages/global-expandable-box/js/index.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+		box.isOpen = false;
+	} else {
+		open(box, closetext);
+		box.isOpen = true;
+	}
+}
+
+function init(boxes, height = '95px', closetext = 'Show less', opentext = 'Show more') {
+	boxes.forEach(box => {
+		let buttonEtc = '<div><div></div><div><button type="button" aria-expanded="false"><span>Show more</span></button></div></div>';
```
Comment (alexkilgour) [inline packages/global-expandable-box/js/index.js]: what's with all the `divs`?

Comment (alexkilgour) [reply packages/global-expandable-box/js/index.js]: also, should `Show more` come from the init param rather then being hardcoded into the button here?

**Pull request merged**